### PR TITLE
#77-Error handling userSearch homePage-Zobeir-Rigi

### PR DESCRIPTION
**Pull request by Zobeir-Rigi**
## Description
Only Cyf user should check their milestone so if you are not in the Cyf clan, it errors and cant go further, also Milestone.js file is modified.
## Related to
[Ticket](https://github.com/BoshraM/ldn9-Ctrl-Shift-Learn/issues/77)
Fixes #1

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/component/User.js [hunk 1]
```diff
@@ -12,17 +11,22 @@ export const User = () => {
...
+					type="text"
+					id="username"
+					onChange={handleInputChange}
+				/>
+				<div>
 				<Search />
-			</Link>
+			</div>
+			</div>
 		</div>
```
Comment (PeteLindsell) [inline client/src/component/User.js:30]: Keep an eye on your formatting, it is difficult to follow where you are closing tags here. If you dont need the wrappinf div you can use a fragment instead <></>

File: client/src/component/Search.js [hunk 2]
```diff
@@ -1,45 +1,62 @@
...
+				return setCodewarsData(data);
 
+			})
+			.catch((err) => {
+				console.error(err);
+                setError(true);
+                });
+                setTimeout(() => {
+                    setError(false);
+                  }, 3000);
```
Comment (PeteLindsell) [inline client/src/component/Search.js:31]: Maybe you should leave the error there until the user starts to type in the field rather than removing after 3 seconds as it will be invalid until there is a change so we dont want to remove it

**Review by PeteLindsell — COMMENTED** [review]
Looks good, a couple of comments and keep an eye on formatting as I think there are formatting issues on all the files which make them hard to follow

**Review by BoshraM — APPROVED** [review]
